### ISSUE TITLE
Add a full date

### ISSUE THREAD

**Issue by chronoss09** [OperationNT414C/ShellSecBat#1]
BenMitnick has add a full date to your plugin. Can you do it and place it to the left or to the center for home menu ?

<a href="http://www.casimages.com/img.php?i=1708200316326637115230447.jpg" target="_blank" title="hebergement d'image"><img src="http://nsm07.casimages.com/img/2017/08/20//1708200316326637115230447.jpg" border="0" alt="hebergement d'image" /></a>

Sorry for my bad english :(

**Comment by OperationNT414C** [comment]
I already answered to those questions here:
http://wololo.net/talk/viewtopic.php?f=116&t=48325&start=10

Unfortunatly, it would require some "SceShell" module disassembling skills and I don't have this kind of skill (I am not a hacker, just a user with some programming skills): all credits for this part goes to "nowrep" for "ShellBat" plugin.
As consequence, I didn't put the full date because it quicky "goes beyond" opened application icons (on your screenshot, you are in "24 hours" format, just imagine if you add "AM" in the time).

I must confess the usage I have for "ShellDateSecBat" is on PS TV where it is more difficult to see the current date (it has no "opening page" when it exits from sleep): the fact that I removed the battery percent allows more space. On my PS Vita, I stick to "ShellSecBat" use because it is more easy to get the current date on this device.
